src/frontend/src/api.js

```diff
@@ -1,3 +1,12 @@
+export function getApiUrl() {
```

Comment:
Trenger vi egentlig denne filen? Kan vi ikke bare kopiere det inn i index.js siden det er snakk om 2 funksjoner.

Reply:
Nei, trenger den ikke. Men har heller ingenting å si siden browserify pakker alt sammen for oss.